src/modules/layout/components/header/user-menu.tsx

```diff
@@ -49,19 +50,19 @@ export function UserMenu() {
         sx={{ mt: 1 }}
       >
         {authStatus === 'authenticated' ? (
-          <>
+          <Box>
```

Comment:
Why box?

Reply:
> Why box?

Because of this error:
![image](https://github.com/user-attachments/assets/ac76c9d7-cefc-4aff-9fc6-da2b7ea4b422)